<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearShiftClass</class>
 <widget class="QWidget" name="GearShiftClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GearShift</string>
  </property>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>94</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>8</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ParkButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReverseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NeutralButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DriveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="ParkIndicator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ReverseIndicator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NeutralIndicator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DriveIndicator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>DriveIndicator</zorder>
  <zorder>NeutralIndicator</zorder>
  <zorder>ReverseIndicator</zorder>
  <zorder>ParkIndicator</zorder>
  <zorder>menuBar</zorder>
  <zorder>mainToolBar</zorder>
  <zorder>statusBar</zorder>
  <zorder>ParkButton</zorder>
  <zorder>ReverseButton</zorder>
  <zorder>NeutralButton</zorder>
  <zorder>DriveButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
